### PR TITLE
Add intro + link descriptions

### PR DESCRIPTION
**Pull request by Dhghomon**
Feel free to take an axe to anything here

### CODE REVIEW

File: docs/cheatsheet/index.rst [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
 
-===========
-Cheatsheets
-===========
+==============================
+Cheatsheets: EdgeDB by example
+==============================
 
-A collection of cheatsheets.
+Just getting started? Keep an eye on this collection of cheatsheets with handy examples for what you'll need to get started with EdgeDB. After familiarizing yourself with them, feel free to dive into more EdgeDB via our longer [interactive tutorial](https://tutorial.edgedb.com/) and **much** longer [Easy EdgeDB textbook](https://website-git-myst.edgedb.vercel.app/easy-edgedb).
```
Comment (1st1) [inline docs/cheatsheet/index.rst:7]: As I suggested in other PR, please reformat to have max 79 characters per line -- that's a linter rule that won't let tests pass.

**Review by elprans — APPROVED** [review]

File: docs/cheatsheet/index.rst [hunk 2]
```diff
@@ -1,33 +1,36 @@
...
-===========
+==============================
+Cheatsheets: EdgeDB by example
+==============================
 
-A collection of cheatsheets.
+Just getting started? Keep an eye on this collection of cheatsheets with 
+handy examples for what you'll need to get started with EdgeDB. 
+After familiarizing yourself with them, feel free to dive into more EdgeDB
+via our longer [interactive tutorial] and **much** longer [Easy EdgeDB textbook].
```
Comment (1st1) [inline docs/cheatsheet/index.rst:10]:
```suggestion
Just getting started? Keep an eye on this collection of cheatsheets with
handy examples for what you'll need to get started with EdgeDB.
After familiarizing yourself with them, feel free to dive into more EdgeDB
via our longer [interactive tutorial] and **much** longer [Easy EdgeDB textbook].
```

File: docs/cheatsheet/index.rst [hunk 3]
```diff
@@ -1,33 +1,36 @@
...
 
 CLI/Admin:
 
-* :ref:`CLI Usage <ref_cheatsheet_cli>`
-* :ref:`REPL <ref_cheatsheet_repl>`
-* :ref:`Administration <ref_cheatsheet_admin>`
+* :ref:`CLI Usage <ref_cheatsheet_cli>` - Getting your database started.
+* :ref:`REPL <ref_cheatsheet_repl>` - Shortcuts for frequently used commands in the EdgeDB REPL.
+* :ref:`Administration <ref_cheatsheet_admin>` - Database and role creation, passwords, port configuration, etc.
```
Comment (1st1) [inline docs/cheatsheet/index.rst:16]:
```suggestion
* :ref:`CLI Usage <ref_cheatsheet_cli>` -- Getting your database started.
* :ref:`REPL <ref_cheatsheet_repl>` -- Shortcuts for frequently used 
  commands in the EdgeDB REPL.
* :ref:`Administration <ref_cheatsheet_admin>` -- Database and role creation, 
  passwords, port configuration, etc.
```

File: docs/cheatsheet/index.rst [hunk 4]
```diff
@@ -1,33 +1,38 @@
...
-* :ref:`UPDATE <ref_cheatsheet_update>`
-* :ref:`DELETE <ref_cheatsheet_delete>`
-* :ref:`GraphQL <ref_cheatsheet_graphql>`
-* :ref:`Special Syntax <ref_cheatsheet_syntax>`
+* :ref:`SELECT <ref_cheatsheet_select>` - Retrieve or compute a set of values.
+* :ref:`INSERT <ref_cheatsheet_insert>`- Create new database objects.
+* :ref:`UPDATE <ref_cheatsheet_update>` - Update database objects.
+* :ref:`DELETE <ref_cheatsheet_delete>` - Remove objects from the database.
+* :ref:`GraphQL <ref_cheatsheet_graphql>` - GraphQL queries supported natively out of the box.
+* :ref:`Special Syntax <ref_cheatsheet_syntax>` - Info on types of strings, number literals, etc.
```
Comment (1st1) [inline docs/cheatsheet/index.rst:27]:
```suggestion
* :ref:`SELECT <ref_cheatsheet_select>` - Retrieve or compute a set of values.
* :ref:`INSERT <ref_cheatsheet_insert>`- Create new database objects.
* :ref:`UPDATE <ref_cheatsheet_update>` - Update database objects.
* :ref:`DELETE <ref_cheatsheet_delete>` - Remove objects from the database.
* :ref:`GraphQL <ref_cheatsheet_graphql>` - GraphQL queries supported natively
  out of the box.
* :ref:`Special Syntax <ref_cheatsheet_syntax>` - Info on types of strings,
  number literals, etc.
```

File: docs/cheatsheet/index.rst [hunk 5]
```diff
@@ -1,33 +1,40 @@
...
-* :ref:`Types <ref_cheatsheet_types>`
-* :ref:`Functions <ref_cheatsheet_functions>`
-* :ref:`Aliases <ref_cheatsheet_aliases>`
-* :ref:`Annotations <ref_cheatsheet_annotations>`
-* :ref:`Migrations <ref_cheatsheet_migrations>`
+* :ref:`Types <ref_cheatsheet_types>` - Make your own object and abstract types on top of existing system types.
+* :ref:`Functions <ref_cheatsheet_functions>` - Write and overload your own strongly typed functions.
+* :ref:`Aliases <ref_cheatsheet_aliases>` - Use aliases to create new types and modify existing ones on the fly.
+* :ref:`Annotations <ref_cheatsheet_annotations>` - Add human readable descriptions to items in your schema.
+* :ref:`Migrations <ref_cheatsheet_migrations>` - Describe your final schema and let EdgeDB put it together for you.
```
Comment (1st1) [inline docs/cheatsheet/index.rst:37]:
```suggestion
* :ref:`Types <ref_cheatsheet_types>` -- Make your own object and abstract
  types on top of existing system types.
* :ref:`Functions <ref_cheatsheet_functions>` -- Write and overload your own
  strongly typed functions.
* :ref:`Aliases <ref_cheatsheet_aliases>` -- Use aliases to create new types
  and modify existing ones on the fly.
* :ref:`Annotations <ref_cheatsheet_annotations>` -- Add human readable
  descriptions to items in your schema.
* :ref:`Migrations <ref_cheatsheet_migrations>` -- Describe your final
  schema and let EdgeDB put it together for you.
```

File: docs/cheatsheet/index.rst [hunk 6]
```diff
@@ -1,33 +1,40 @@
...
-* :ref:`GraphQL <ref_cheatsheet_graphql>`
-* :ref:`Special Syntax <ref_cheatsheet_syntax>`
+* :ref:`SELECT <ref_cheatsheet_select>` - Retrieve or compute a set of values.
+* :ref:`INSERT <ref_cheatsheet_insert>`- Create new database objects.
+* :ref:`UPDATE <ref_cheatsheet_update>` - Update database objects.
+* :ref:`DELETE <ref_cheatsheet_delete>` - Remove objects from the database.
+* :ref:`GraphQL <ref_cheatsheet_graphql>` - GraphQL queries supported natively
+  out of the box.
+* :ref:`Special Syntax <ref_cheatsheet_syntax>` - Info on types of strings,
+  number literals, etc.
```
Comment (1st1) [inline docs/cheatsheet/index.rst:29]:
```suggestion
* :ref:`SELECT <ref_cheatsheet_select>` -- Retrieve or compute a set of values.
* :ref:`INSERT <ref_cheatsheet_insert>` -- Create new database objects.
* :ref:`UPDATE <ref_cheatsheet_update>` -- Update database objects.
* :ref:`DELETE <ref_cheatsheet_delete>` -- Remove objects from the database.
* :ref:`GraphQL <ref_cheatsheet_graphql>` -- GraphQL queries supported natively
  out of the box.
* :ref:`Special Syntax <ref_cheatsheet_syntax>` -- Info on types of strings,
  number literals, etc.
```

File: docs/cheatsheet/index.rst [hunk 7]
```diff
@@ -1,33 +1,45 @@
...
-===========
+==============================
+Cheatsheets: EdgeDB by example
+==============================
 
-A collection of cheatsheets.
+Just getting started? Keep an eye on this collection of cheatsheets with
+handy examples for what you'll need to get started with EdgeDB.
+After familiarizing yourself with them, feel free to dive into more EdgeDB
+via our longer [interactive tutorial] and **much** longer [Easy EdgeDB textbook].
```
Comment (1st1) [inline docs/cheatsheet/index.rst:10]:
```suggestion
Just getting started? Keep an eye on this collection of cheatsheets with
handy examples for what you'll need to get started with EdgeDB.
After familiarizing yourself with them, feel free to dive into more EdgeDB
via our longer `interactive tutorial <https://tutorial.edgedb.com>`_ and 
**much** longer `Easy EdgeDB textbook </easy-edgedb>`_.
```

**Pull request merged**